src/i18n/en.js

```diff
@@ -179,32 +179,30 @@ export default {
         <strong>Legitimation:</strong> Consent of the applicant.
       </p>
       <p>
-        <strong>Recipients:</strong> Your data will be saved by Doppler, Zoho as CRM, Google as the
-        provider of reCAPTCHA service, Digital Ocean, Cogeco Peer1 and Rackspace as hosting companies.
+        <strong>Recipients:</strong> Your data will be saved by Doppler, Zoho as CRM,
+        Digital Ocean, Cogeco Peer1 and Rackspace as hosting companies.
       </p>
       <p>
         <strong>Additional information:</strong> In Doppler's
         <a target="_blank" href="${urlPrivacyWithQueryString}">Privacy Policy</a>
-        you'll find additional information about the data storage and use of your personal
-        information, including information on access, conservation, rectification, deletion,
-        security, cross-border data transfers and other issues.
+        you’ll find additional information about the data storage and use of your
+        personal information, including information on access, conservation, rectification,
+        deletion, security, cross-border data transfers and other issues.
       </p>
       `,
     log_in: `Log In`,
     no_more_resend: `You haven't received the Email yet? We have already forwarded it to you, if it doesn't arrive in the next few minutes, please contact support`,
     placeholder_email: `Your Email will be your Username`,
```

Comment:
```suggestion
    placeholder_email: `Your Email will be your username`,
```

Reply:
Hi @GusBaamonde ,

I have detected the following discrepancy:

Document
![image](https://user-images.githubusercontent.com/8885635/57457934-dd71a300-7246-11e9-9fe8-740989e1dfdc.png)

Webapp URL
![image](https://user-images.githubusercontent.com/8885635/57458017-009c5280-7247-11e9-990d-730c3c26e748.png)

I have confirmed this with Caro.

Regards

Reply:
I add another content to update:

Document:
![image](https://user-images.githubusercontent.com/8885635/57459188-19a60300-7249-11e9-984e-9f1306cde937.png)

Webapp URL:
![image](https://user-images.githubusercontent.com/8885635/57459258-41956680-7249-11e9-9da5-d2e780cdb143.png)

Regards

Reply:
Tiene que ser `Your Email will be your Username` (con mayusculas)